webapp/src/marked.tsx

```diff
@@ -287,6 +287,16 @@ export class MarkedContent extends data.Component<MarkedContentProps, MarkedCont
 
 
     private renderVideo(content: HTMLElement){
+
+        pxt.Util.toArray(content.querySelectorAll('iframe.yt-embed'))
+        .forEach((inlineVideo: HTMLElement)=>{
+
+            let lang=pxt.appTarget.appTheme?.defaultLocale ?? "en";
+            let src=content.querySelectorAll('iframe.yt-embed')[0].getAttribute('src');
```

Comment:
this looks wrong? I think you want 
```suggestion
            const src = inlineVideo.getAttribute('src');
```

? This will currently only ever get the src of the first video so if you have multiple youtube videos in the tutorial I think they'll all point to the same thing?

Reply:
Good catch, yeah shouldn't have assumed there would be just one video. Thanks!